docs/data/multiple_files/overview.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docu
+title: Reading Multiple Files
+redirect_from:
+  - /docs/data/csv/multiple_files
+---
+
+DuckDB can read multiple files of different types (CSV, Parquet, JSON files) at the same time using either the glob syntax, or by providing a list of files to read. See the [combining schemas][combining_schemas] page for tips on reading files with different schemas.
+
+### CSV
+
+```sql
+-- read all files with a name ending in ".csv" in the folder "dir"
+SELECT * FROM 'dir/*.csv';
+-- read all files with a name ending in ".csv", two directories deep
+SELECT * FROM '*/*/*.csv';
+-- read the CSV files 'flights1.csv' and 'flights2.csv'
+SELECT * FROM read_csv_auto(['flights1.csv', 'flights2.csv'])
+-- read the CSV files 'flights1.csv' and 'flights2.csv', unifying schemas by name and outputting a `filename` column
+SELECT * FROM read_csv_auto(['flights1.csv', 'flights2.csv'], union_by_name=True, filename=True)
+```
+
+### Parquet
+
+```sql
+-- read all files that match the glob pattern
+SELECT * FROM 'test/*.parquet';
+-- read 3 parquet files and treat them as a single table
+SELECT * FROM read_parquet(['file1.parquet', 'file2.parquet', 'file3.parquet']);
+-- Read all parquet files from 2 specific folders
+SELECT * FROM read_parquet(['folder1/*.parquet','folder2/*.parquet']);
+```
+
+
+### Multi-File Reads and Globs
+DuckDB can also read a series of Parquet files and treat them as if they were a single table. Note that this only works if the Parquet files have the same schema. You can specify which Parquet files you want to read using a list parameter, glob pattern matching syntax, or a combination of both.
+
+#### List Parameter
+The read_parquet function can accept a list of filenames as the input parameter.
+
+```sql
+-- read 3 parquet files and treat them as a single table
+SELECT * FROM read_parquet(['file1.parquet', 'file2.parquet', 'file3.parquet']);
+```
+
+#### Glob Syntax
+Any file name input to the read_parquet function can either be an exact filename, or use a glob syntax to read multple files that match a pattern.
+
+|  Wildcard  |                        Description                        |
+|------------|-----------------------------------------------------------|
+| `*`        | matches any number of any characters (including none)     |
+| `?`        | matches any single character                              |
```

Comment:
Maybe we should mention here that we recently disabled the ? in globs for S3 urls

Reply:
Added a note, let me know if you want to change anything about it

Reply:
perfect!